src/net.cpp

```diff
@@ -1135,6 +1135,9 @@ void CConnman::DisconnectNodes()
                 // close socket and cleanup
                 pnode->CloseSocketDisconnect();
 
+                // update connection count by network
+                if (pnode->IsManualOrFullOutboundConn()) m_network_conn_counts[pnode->ConnectedThroughNetwork()]--;
```

Comment:
here and in `test/util/net.h` - this is still using `ConnectedThroughNetwork` (see previous comment).